akka-http-core/src/main/scala/akka/http/impl/engine/ws/FrameOutHandler.scala

```diff
@@ -44,6 +43,9 @@ private[http] class FrameOutHandler(serverSide: Boolean, _closeTimeout: FiniteDu
       case UserHandlerCompleted ⇒
         become(new WaitingForPeerCloseFrame())
         ctx.push(FrameEvent.closeFrame(Protocol.CloseCodes.Regular))
+      case UserHandlerErredOut(ex) ⇒
+        become(new WaitingForPeerCloseFrame())
+        ctx.push(FrameEvent.closeFrame(Protocol.CloseCodes.UnexpectedCondition, "internal error"))
```

Comment:
@sirthias @jrudolph please take a look, I think this was missing but I’m not sure about what exactly should be done in this case